Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -106,5 +110,18 @@ public async Task RemoveResponse(DiscordRequest request, Contexts contexts)
             await _responsesService.RemoveResponse(onEvent, contexts.Server.Id);
             await messageService.SendResponse(x => x.ResponseHasBeenRemoved(contexts, onEvent), contexts);
         }
+
+        [AdminCommand]
+        [DiscordCommand("responses")]
+        public async Task Responses(DiscordRequest request, Contexts contexts)
+        {
+            var possibleArguments = new string[] { "all", "default", "custom" };
+            var argument = request.Arguments?.FirstOrDefault()?.Name?.ToLowerInvariant();
```

Comment:
```suggestion
            var argument = request.FirstArgument?.ToLowerInvariant();
```

Reply:
Wtedy by `FirstArgument` byłoby właściwością klasy DiscordRequest

Reply:
co masz na myśli?
jeśli chodzi ci o dodanie propertiesa
to aktualny sposób to stara wersja od której odchodzimy
ale w części przypadków (jak np ten) nie jesteśmy w stanie zrobić pewnych rzeczy nową wersją

(jeszcze, stopniowo to sie zmienia)